src/Stryker.CLI/Stryker.CLI/OptionsBuilder.cs

```diff
@@ -63,37 +71,51 @@ public class OptionsBuilder
                 thresholdLow: GetOption(thresholdLow.Value(), CLIOptions.ThresholdLow),
                 thresholdBreak: GetOption(thresholdBreak.Value(), CLIOptions.ThresholdBreak),
                 filesToExclude: GetOption(filesToExclude.Value(), CLIOptions.FilesToExclude),
+                filePatterns: GetOption(filePatterns.Value(), CLIOptions.FilePatterns),
                 testRunner: GetOption(testRunner.Value(), CLIOptions.TestRunner),
                 solutionPath: GetOption(solutionPath.Value(), CLIOptions.SolutionPath),
                 languageVersion: GetOption(languageVersion.Value(), CLIOptions.LanguageVersionOption));
         }
 
         private T GetOption<V, T>(V cliValue, CLIOption<T> option)
         {
+            T value = default(T);
+            var hasValue = false;
+
             if (cliValue != null && ((option.ValueType == CommandOptionType.NoValue && cliValue is bool boolValue && boolValue == true) || option.ValueType != CommandOptionType.NoValue))
             {
                 // Convert the cliValue string to the desired type
-                return ConvertTo(cliValue, option);
+                value  = ConvertTo(cliValue, option);
+                hasValue = true;
             }
-            else if (config != null)
+            else if (_config != null)
             {
                 // Try to get the value from the config file
                 if (typeof(IEnumerable).IsAssignableFrom(typeof(T)) && typeof(T) != typeof(string))
                 {
-                    return config.GetSection(option.JsonKey).Get<T>();
+                    value = _config.GetSection(option.JsonKey).Get<T>();
+                    hasValue = true;
                 }
                 else
                 {
-                    string configValue = config.GetValue(option.JsonKey, string.Empty).ToString();
+                    string configValue = _config.GetValue(option.JsonKey, string.Empty).ToString();
                     if (!string.IsNullOrEmpty(configValue))
                     {
-                        return ConvertTo(configValue, option);
+                        value = ConvertTo(configValue, option);
+                        hasValue = true;
                     }
                 }
             }
 
             // Unable to get value from user, return default value
-            return option.DefaultValue;
+            if (!hasValue)
```

Comment:
brackets.

Feel free to add that .editorconfig if that helps you keep resharper in check.